conans/cli/commands/install.py

```diff
@@ -143,7 +143,8 @@ def install(conan_api, parser, *args):
     parser.add_argument("-of", "--output-folder",
                         help='The root output folder for generated and build files')
     parser.add_argument("-sf", "--source-folder", help='The root source folder')
-
+    parser.add_argument("--deploy", action=Extender,
+                        help='Deploy using the provided deployer to the output folder')
```

Comment:
I think we should add examples of possible valid `deployer` values, e.g., `conan_full_deploy`, `mydeploy.py`, `my/path/to/my/deployer`, etc.

Reply:
Indeed, you can specify it in the docs, but I think it could be important to put a couple of examples here just to demonstrate what are the most common values.